processor/agentmetricsprocessor/factory_test.go

```diff
@@ -21,15 +21,16 @@ import (
 	"github.com/stretchr/testify/assert"
 	"go.opentelemetry.io/collector/component/componenttest"
 	"go.opentelemetry.io/collector/consumer/consumertest"
-	"go.opentelemetry.io/collector/processor"
+	"go.opentelemetry.io/collector/processor/processortest"
 )
 
 func TestCreateDefaultConfig(t *testing.T) {
 	assert.NoError(t, componenttest.CheckConfigStruct(NewFactory().CreateDefaultConfig()))
 }
 
 func TestCreateProcessor(t *testing.T) {
-	mp, err := createMetricsProcessor(context.Background(), processor.CreateSettings{}, createDefaultConfig(), consumertest.NewNop())
+	cfg := createDefaultConfig().(*Config)
```

Comment:
Could you explain why this was done? I'm having trouble understanding the reason.

Reply:
@braydonk Honestly I can't remember why I did this for the config...but good catch! Doesn't seem to be necessary. Might have came from a copy/paste while looking at existing factory tests within the latest release.

Pushing a change to just pass in `createDefaultConfig()` in the 3 files where I did this.